docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -768,7 +768,8 @@ Start by adding the following dependencies to your test project:
 </dependency>
 ----
 
-[[integration-testing-wiremock]]
+ifndef::no-deprecated-test-resource[]
```

Comment:
Like in bearer and code flow docs, this option is important if testing against Keycloak Dev Service does not suit